Error fetching image manifest list from private AWS ECR repo

I have been trying to use `enroot import` with a private AWS ECR repository, and have been running into an issue when retrieving the image manifest list. I was able to setup my `.credentials` file and successfully complete the authentication step following [NVIDIA/enroot#59](https://github.com/NVIDIA/enroot/issues/59), however when fetching the image manifest list I got the following error:
```
$ enroot import docker://0000.dkr.ecr.us-east-2.amazonaws.com/image-repo:image-tag
[INFO] Querying registry for permission grant
[INFO] Authenticating with user: AWS
[INFO] Using credentials from file: /path/to/enroot/.credentials
[INFO] Authentication succeeded
[INFO] Fetching image manifest list
[ERROR] Could not process JSON input
```
This error is being thrown by the [`common::jq`](https://github.com/NVIDIA/enroot/blob/9c6e979059699e93cfc1cce0967b78e54ad0e263/src/common.sh#L139) function being called in [`docker::_download`](https://github.com/NVIDIA/enroot/blob/9c6e979059699e93cfc1cce0967b78e54ad0e263/src/docker.sh#L147), and when printing the curl response that is being passed into `common::jq`, I found that the response to the manifest list request was `Not Authorized` in plain text. This error has been thrown both when installing enroot from source and from packages.

Do you have any insight into what might be causing this issue? 

Enroot version: 3.4.0
Docker version: 20.10.17

--- a/src/docker.sh
+++ b/src/docker.sh
@@ -73,7 +73,7 @@ docker::_authenticate() {
         ;;
     Basic)
         # Check that we have valid credentials and save them if successful.
-        CURL_ERROUT=1 common::curl "${curl_opts[@]}" -G -v ${req_params[@]+"${req_params[@]}"} -- "${url}" \
+        CURL_ERROUT=1 common::curl "${curl_opts[@]}" -w -G -v ${req_params[@]+"${req_params[@]}"} -- "${url}" \
           | awk '/Authorization: Basic/ { sub(/\r/, "", $4); print $4 }' \
           | common::read -r token
         ;;

